src/main/java/com/example/pfairplayservice/jpa/model/MemberEntity.java

```diff
@@ -29,39 +35,46 @@
     private String uid;
 
     @Column(nullable = false)
+    @Length(min = 6, max = 10)
     private String id;
 
     @Column(nullable = false)
+    @Length(min = 6, max = 10)
     private String password;
 
     @Column(nullable = false)
+    @Length(min = 2, max = 10)
     private String name;
 
     @Column(nullable = false)
+    @Temporal(TemporalType.TIMESTAMP)
     private Date birthday;
 
     @Column(nullable = false)
     private String address;
 
-    @Column(nullable = false)
+    @Column(nullable = false, length = 11)
     private String phoneNumber;
 
-    @Column
+    @Column(length = 1)
     private Integer preferPosition;
 
-    @Column
+    @Column(length = 1)
+    @ColumnDefault("1")
     private Integer level;
 
-    @Column(nullable = false)
+    @Column(nullable = false, length = 1)
     @ColumnDefault("1")
     private Integer phoneNumberDisclosureOption;
 
-    @CreationTimestamp
     @Column(nullable = false)
+    @CreationTimestamp
+    @Temporal(TemporalType.TIMESTAMP)
```

Comment:
Why do you use `TIMESTAMP` in MySQL?
Do you need a milliseconds unit for this field?
If it does not, Date type in MySQL is better.

Reply:
Right. Timestamp type is unnecessary. 
When you comment Timestamp on https://github.com/dgryoo/pfairplayService/pull/20#discussion_r579897114 , 
i was interested in that. So i apply. That was good experience.
But now  i will change Timestamp to Date.